### DISCUSSION TITLE
nested containers

### DISCUSSION THREAD

**Discussion by tadmi** [vuepress-theme-hope/vuepress-theme-hope#2855] [Q&A] [answered]
Please add the possibility of nested containers

::: details
   ::: details
   :::
:::

**Comment by Mister-Hope** [comment, marked as answer]
Please learn some basics: https://mdit-plugins.github.io/container.html

Also the docs are nesting in several places: https://plugin-md-enhance.vuejs.press/guide/align.html

**Comment by tadmi** [comment]
It doesn't work properly:

:::: details details
::: details details
text
:::
::::

https://user-images.githubusercontent.com/63556979/222494738-f10172f6-a4a4-4d7b-8708-6a80385e8e33.mp4

**Reply by Mister-Hope** [reply]
FIxed in https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/2f1743b5faad89577a448eff5f53e3247a87b459
